ref/__init__.py

```diff
@@ -1,7 +1,7 @@
 """To use this module just use the main function at the top of your code"""
 from types import ModuleType
 import importlib
-__protectfiles, __protectdirs, _lockperms = None, None,None
+__protectfiles, __protectdirs, __lockperms = None, None, None
```

Comment:
This seems to fix the problem. 
Sorry for the typo

Reply:
This isn't the mistake, I fixed this too, using `_` and not `__` make this variable just protected and not hidden